tests/test_methods.py

```diff
@@ -103,16 +103,14 @@ def test_sgpr_qu():
 
     @tf.function
     def closure():
-        return - model.log_likelihood()
+        return model.neg_log_marginal_likelihood()
 
     gpflow.optimizers.Scipy().minimize(closure, variables=model.trainable_variables)
+    [qu_m, qu_A] = model.compute_qu()
+    [f_atZ_m, f_atZ_A] = model.predict_f(model.inducing_variable.Z, full_cov=True)
 
-    q_mu = model.mean_function(model.inducing_variable.Z.read_value())
-    q_var = model.kernel(model.inducing_variable.Z.read_value(), full=True)
-    model_qu = [q_mu, tf.reshape(q_var, (1, 20, 20))]
-
-    for v1, v2 in zip(model_qu, model.predict_f(model.inducing_variable.Z.read_value(), full_cov=True)):
-        np.testing.assert_allclose(v1, v2, rtol=0, atol=1e-7)
+    np.testing.assert_allclose(qu_m, f_atZ_m, rtol=0, atol=1e-5)
+    np.testing.assert_allclose(tf.reshape(qu_A, (1, 20, 20)), f_atZ_A, rtol=0, atol=1e-5)
```

Comment:
can the tolerance be any tighter? maybe trading off a bit of rtol ~ 1e-7 against atol ~ 1e-7 or so if needed?

Reply:
```suggestion
    np.testing.assert_allclose(tf.reshape(qu_A, (1, 20, 20)), f_atZ_A, rtol=0, atol=1e-6)
```